sentry-android-ndk/src/main/jni/sentry.c

```diff
@@ -11,8 +11,257 @@ struct transport_options {
 
 struct transport_options g_transport_options;
 
+JNIEXPORT void JNICALL
```

Comment:
we don't need to check NPEs here, because on Java the thread-safe list dont accept null keys and values